sdks/go/pkg/beam/core/graph/fn.go

```diff
@@ -196,13 +217,31 @@ func (f *DoFn) Name() string {
 
 // IsSplittable returns whether the DoFn is a valid Splittable DoFn.
 func (f *DoFn) IsSplittable() bool {
-	return false // TODO(BEAM-3301): Implement this when we add SDFs.
+	isSdf, _ := validateSdfMethodsPresent((*Fn)(f))
```

Comment:
Good point, I'll keep it in mind for the future. For this one, though, I realized it's better just to check for the presence of a single SDF method anyway, rather than calling that validation method.